### PR TITLE
https://issues.redhat.com/browse/ACM-7341 Retry update for mirror amq-stream operator

### PR DESCRIPTION
**Pull request by dockerymick**
https://issues.redhat.com/browse/ACM-7341

### CODE REVIEW

File: global_hub/global_hub_disconnected.adoc [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+[#global-hub-packages-in-catalog]
+=== Creating operator packages in your catalog
+
+Red Hat provides the multicluster global hub and AMQ Streams operators in the Red Hat operators catalog, which are delivered by the `registry.redhat.io/redhat/redhat-operator-index` index image. When you prepare your mirror of this catalog index image, you can choose to either mirror the entire catalog as provided by Red Hat, or you can mirror a subset that contains only the operator packages that you intend to use.
+
+If you are creating a full mirror catalog, no special considerations are needed as all of the packages required to install multicluster global hub and AMQ Streams are included. However, if you are creating a partial or filtered mirrored catalog, for which you identify particular packages to be included, you must to include the `multicluster-global-hub-operator-product` and `amq-streams` package names in your list. 
+
+
+//Mirror the catalog images to the registry with one of the following procedures:
+//which procedure is easiest for the user? Is it necessary to have both of the procedures?
```
Comment (yanmxa) [inline global_hub/global_hub_disconnected.adoc]: I have tried the second method and it works! The first one seems simpler. We can keep both. 

@hchenxa What's your opinion?

File: global_hub/global_hub_disconnected.adoc [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
++
+[source,bash]
+----
+oc mirror --config=./imageset-config.yaml docker://myregistry.example.com:5000
+----
+
+. Mirror the image set in a fully disconnected environment. For more details, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html/installing/disconnected-installation-mirroring#installing-mirroring-installation-images[Mirroring images for a disconnected installation].
+
+//do you mean command line instead of utility?
+* Create the mirrored catalog or registry by using the `opm` utility. Build and push the multicluster global hub index image:
```
Comment (yanmxa) [inline global_hub/global_hub_disconnected.adoc]: 1. Both methods use the command line.  

2. Remove the content `Build and push the multicluster global hub index image:` from this line.

File: global_hub/global_hub_disconnected.adoc (hunk 1 above)
Comment (dockerymick) [reply global_hub/global_hub_disconnected.adoc]: Will you describe when or why users will need both methods?  My opinion is to keep the simple one

File: global_hub/global_hub_disconnected.adoc (hunk 2 above)
Comment (dockerymick) [reply global_hub/global_hub_disconnected.adoc]: is utility another word for command line?

File: global_hub/global_hub_disconnected.adoc [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+----
+$ oc get pods -n multicluster-global-hub
+NAME                                                READY   STATUS    RESTARTS   AGE
+multicluster-global-hub-operator-687584cb7c-fnftj   1/1     Running   0          2m12s
+----  
+
+[#additional-resources-disconnected]
+== Additional resources
+
+- link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html/installing/disconnected-installation-mirroring#creating-mirror-registry[Create a mirror registry]
```
Comment (dockerymick) [inline global_hub/global_hub_disconnected.adoc:289]: Chris will ask what OCP versions this was tested on

Comment (dockerymick) [inline global_hub/global_hub_disconnected.adoc]: change to 4.12 bc it was tested on 4.12

File: global_hub/global_hub_disconnected.adoc [hunk 4]
```diff
@@ -0,0 +1,292 @@
...
+- link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html/installing/disconnected-installation-mirroring#installing-mirroring-installation-images[Mirroring images for a disconnected installation]
+- that are described in the topic link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html-single/operators/index#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks]
+- link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html-single/operators/index#olm-restricted-networks-operatorhub_olm-restricted-networks[Disabling the default OperatorHub sources]
```
Comment (dockerymick) [inline global_hub/global_hub_disconnected.adoc]:
```suggestion
- link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/installing/disconnected-installation-mirroring#creating-mirror-registry[Create a mirror registry]
- link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/installing/disconnected-installation-mirroring#installing-mirroring-installation-images[Mirroring images for a disconnected installation]
- that are described in the topic link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html-single/operators/index#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks]
- link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html-single/operators/index#olm-restricted-networks-operatorhub_olm-restricted-networks[Disabling the default OperatorHub sources]
```

File: global_hub/global_hub_overview.adoc [hunk 5]
```diff
@@ -1,33 +1,51 @@
...
 
-    - xref:../global_hub/global_hub_upgrading.adoc#global-hub-upgrading[Upgrading multicluster global hub]
-    -xref:../global_hub/global_hub_uninstalling.adoc#global-hub-uninstalling[Uninstalling multicluster global hub]
+[%autowidth]
+.Table of network requirements for the managed hub cluster
+|===
+|Direction | Protocol | Connection | Port (if specified) | Source address |	Destination address
 
+|Outbound to Kafka Cluster | HTTPS | Global hub agent need to sync cluster info and policy info to Kafka cluster | 443 | multicluster-global-hub-agent pod | Kafka route host
+|===
```
Comment (dockerymick) [inline global_hub/global_hub_overview.adoc]:
```suggestion
| Outbound to Kafka Cluster 
| HTTPS 
| Global hub agent need to sync cluster info and policy info to Kafka cluster 
| 443 
| multicluster-global-hub-agent pod 
| Kafka route host
|===
```

File: global_hub/global_hub_disconnected.adoc [hunk 6]
```diff
@@ -0,0 +1,292 @@
...
+      imageURL: myregistry.example.com:5000/mirror/oc-mirror-metadata
+mirror:
+  platform:
+    channels:
+    - name: stable-4.12
+      type: ocp
+  operators:
+  - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.12
+    packages:
+    - name: multicluster-global-hub-operator-product
```
Comment (yanmxa) [inline global_hub/global_hub_disconnected.adoc]: @dockerymick In this PR, can you rename the package `multicluster-global-hub-operator-product` with `multicluster-global-hub-operator-rh` globally? Cause we renamed it recently.

File: global_hub/global_hub_disconnected.adoc (hunk 2 above)
Comment (yanmxa) [reply global_hub/global_hub_disconnected.adoc]: Sorry for confusing you with my words. Yes! the utility and command line means the same thing here.

File: global_hub/global_hub_disconnected.adoc (hunk 1 above)
Comment (yanmxa) [reply global_hub/global_hub_disconnected.adoc]: Okay! Let's make it simpler. Just keep the first one - using the `oc-mirror` plug-in

File: global_hub/global_hub_disconnected.adoc (hunk 2 above)
Comment (dockerymick) [reply global_hub/global_hub_disconnected.adoc]: No worries! Thanks for clarifying !

File: global_hub/global_hub_disconnected.adoc (hunk 6 above)
Comment (dockerymick) [reply global_hub/global_hub_disconnected.adoc]: Done

File: global_hub/global_hub_release_notes.adoc [hunk 7]
```diff
@@ -6,6 +6,6 @@ Learn about the current release.
 * xref:../global_hub/global_hub_known_issues.adoc#known-issues-global-hub[Multicluster global hub known issues]
 * xref:../global_hub/global_hub_architecture.adoc#global-hub-architecture[Multicluster global hub architecture]
 
-If you experience issues with one of the currently supported releases, or the product documentation, go to link:https://www.redhat.com/en/services/support[Red Hat Support] where you can troubleshoot, view Knowledgebase articles, connect with the Support Team, or open a case. You must log in with your credentials.
+To view the supported versions for multicluster global hub, see the link:https://access.redhat.com/articles/7033110[Multicluster global hub support matrix]. If you experience issues with one of the currently supported releases, or the product documentation, go to link:https://www.redhat.com/en/services/support[Red Hat Support] where you can troubleshoot, view Knowledgebase articles, connect with the Support Team, or open a case. You must log in with your credentials.
```
Comment (dockerymick) [inline global_hub/global_hub_release_notes.adoc:9]: Global hub doesn't have a separate SKU, support matrix is not needed but the info should be doc in the product doc (suggested by Brandi and bc it's late in the release

**Comment by dockerymick** [comment]
I will continue editing this PR after [@cadawson PR](https://github.com/stolostron/rhacm-docs/pull/5699/files) is merged

**Comment by cadawson** [comment]
Merged in outside of this PR. Conflicts cause me to close this, but @yanmxa 's and @dockerymick 's hard work live on in the doc. Thanks!

**Pull request closed without merging**